### PR TITLE
smash-pumpkins.lic: Various updates.

### PR DESCRIPTION
**Pull request by Sarvatt**
Use the fix_standing helper from common.
Check for not enough coins to continue and withdraw from the bank, or exit if not enough coin.
switch an fput to a bput to put the item in the bucket
change the safe room to the lobby now that the entrance is magic free in 2017.

### CODE REVIEW

**Comment by Sarvatt** [comment]
I found a few more fixes I might as well squeeze in here, please don't pull just yet

**Comment by Sarvatt** [comment]
There we go. The timing on the second tend is iffy, 20 seconds seems to catch it every time though.

File: smash-pumpkins.lic [hunk 1]
```diff
@@ -1,48 +1,67 @@
...
+    DRCI.stow_hands
+  end
   pause
 
   DRCT.walk_to(10_570)
   while thing = DRC.left_hand || DRC.right_hand
     case thing
     when *junk_list
-      fput("put #{thing} in bucket")
+      DRC.bput("put #{thing} in bucket", 'You drop')
```
Comment (rpherbig) [inline smash-pumpkins.lic]: Does `dispose_trash` do what you need?

File: smash-pumpkins.lic [hunk 2]
```diff
@@ -1,48 +1,67 @@
...
     else
       # if thing =~ /ticket/
```
Comment (rpherbig) [inline smash-pumpkins.lic]: Can this be deleted?

File: smash-pumpkins.lic (hunk 1 above)
Comment (Sarvatt) [reply smash-pumpkins.lic]: Tried changing it to DRCI.dispose_trash(thing) but it just does this
[smash-pumpkins]>put my tattered scarf in bucket
You drop a tattered scarf in a dented metal bucket.
>
[smash-pumpkins]>stow tattered scarf
Stow what?  Type 'STOW HELP' for details.
...
[smash-pumpkins: checked against [/You put/i, /to fit in the/i]]
[smash-pumpkins: for command stow skeleton earrings]

File: smash-pumpkins.lic (hunk 2 above)
Comment (Sarvatt) [reply smash-pumpkins.lic]: done

File: smash-pumpkins.lic (hunk 1 above)
Comment (rpherbig) [reply smash-pumpkins.lic]: That looks like what I'd expect to happen. Line 46 does a 'put' and line 47 does a 'stow'.

Comment (Sarvatt) [reply smash-pumpkins.lic]: I see the problem, I removed the else by mistake, they weren't both meant to be run. Fixed up now.

Comment (rpherbig) [reply smash-pumpkins.lic]: Cool. I'm still in favor of `dispose_trash`, but it isn't a big deal.

**Pull request merged**